medium/users/services.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth.hashers import make_password
+from django.contrib.auth.password_validation import validate_password
+from django.core.validators import validate_email
+
+from medium.users.models import User
+
+
+class UserService:
+    def create_user(self, name: str, email: str, password: str) -> User:
```

Comment:
stick to name convection `<entity>_<action>`
refer to :
https://github.com/HackSoftware/Django-Styleguide#naming-convention

Reply:
Firstly, this reference is not a Quran, so we won't need to stick to every single word in it.
Secondly, the class-based services already achieve the `Namespacing` and `Greppability` that are mentioned in the reference, so we don't need to create a problem and then solve it using a weird naming.

Reply:
Of course, we just need to agree on which route to take to remain consistent across the project.
I just mentioned the guide here as others may haven't which route you took and decided to follow the guide🤷‍♂️